src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/ECDsaX509SignatureGenerator.cs

```diff
@@ -104,7 +113,7 @@ protected override PublicKey BuildPublicKey()
             
             return new PublicKey(
                 ecPublicKey,
-                new AsnEncodedData(ecPublicKey, DerEncoder.EncodeOid(curveOid)),
+                new AsnEncodedData(ecPublicKey, curveOidEncoded),
```

Comment:
not seeing any gain from this change - previous code was shorter though

Reply:
Sorry, that is part of the squashed commits from different PR (#31602). That PR is part of series of patches that remove DerEncoder and DerSequenceReader in favor of AsnReader / AsnWriter / AsnSerializer.

Reply:
fair enough, this specific case would make sense to replace internals of DerEncoder.EncodeOid to use AsnWriter